VERSIONING.md

```diff
@@ -0,0 +1,33 @@
+# About Versioning
+
+This guidelines apply to versioning of docker images in mailserver2 repos:
+
+- mailserver
+- debian-mail-overlay
+- postfixadmin
+- rainloop
+
+Historically the old [hardware/debian-mail-overlay](https://hub.docker.com/r/hardware/debian-mail-overlay/tags) image seemed to be versioned after `rspamd` it includes. However, apart of `rspamd` the image contains a lot of other differently versioned software. In this regard, it's more logical to have an independent version, and note down the version of individual software component in the commit messages and/or release history. 
+
+For `rainloop` and `postfixadmin` it makes sense to continue to version the images after the respective software versions, since it's not likely that they may require a version bump for any other reason.
+
+In certain case it's also possible for the `Dockerfile` itself to change without individual components updates. Finally, it could be that the `Dockerfile` does not change either, but the change is effected by update in an upstream software repository, for example `clamav` updates in `mailserver` are handled this way.
+
+Follow these steps when an update is ready to be pushed to Docker Hub:
+
+- If the only change in an upstream repo, create a commit describing that change, e.g. `git commit --allow-empty -m "update from upstream software repositories"`
+- Tag the current GitHub commit in the `mailserver2` repo with the next version
```

Comment:
I think this should be done with annotated tags.
e.g. `git tag -a v1.0.1 -m "update postfixadmin to version 1.0.1, ... other changes here"`
I'm wondering if this is enough and can function as a 'changelog'. What about a CHANGELOG.md with the exact changes between each version?

Reply:
I don't mind annotated tags, I'll make the change. As far as changelog goes, I'd prefer to have it, but I do not want to complicate life for others. I do agree that commit history can serve as the changelog for this particular project, especially if we take some care to maintain clean commit history, so if you decide that a changelog is overkill, I won't mind.

Reply:
I've made the proposed change.